setup.cfg

```diff
@@ -110,6 +110,15 @@ ignore_errors = False
 [mypy-glotaran.deprecation.*]
 ignore_errors = False
 
+[mypy-glotaran.optimization.*]
+ignore_errors = False
+
+[mypy-glotaran.optimization.test.*]
+ignore_errors = True
+
+[mypy-glotaran.project.generators.test.*]
+ignore_errors = True
+
```

Comment:
This should be reactivated see [commit](https://github.com/glotaran/pyglotaran/commit/7b1d50725a0c7b31de04d018ab1654a496cde4c6)